src/app/donation-start/donation-start-form/donation-start-form-new.component.html

```diff
@@ -102,6 +101,53 @@
           of main donation
           </mat-hint>
         </div>
+        <app-donation-tipping-slider
+            *ngIf="!showCustomTipInput"
+            #donationTippingSlider
+            [percentageStart]="1"
+            [percentageEnd]="maximumTipPercentage"
+            [donationAmount]="donationAmount"
```

Comment:
Shouldn't this be something like

```suggestion
            [donationAmount]="tipAmount()"
```

Within the tip slider I think donation amount refers to the amount they're tipping, not the amount they're donating to the campaigning charity.

Reply:
ah no I was wrong, ignore me.